### DISCUSSION TITLE
blog/dev/2024-05-07-SMTP-spring-boot/

### DISCUSSION THREAD

**Discussion by giscus** [mingdodev/mingdodev.github.io#15] [Announcements]
# blog/dev/2024-05-07-SMTP-spring-boot/

moonshot

https://mingdodev.github.io/blog/dev/2024-05-07-SMTP-spring-boot/

**Comment by mikio999** [comment]
SMTP의 포트번호는 25입니다!

**Reply by mingdodev** [reply]
!!!!!!!!!🥺 네트워크 공부한 사람 너무 좋아요

**Reply by mingdodev** [reply]
아!!!!!! 아!! SMTP도 자세히 다루지 않았는데 변수 설명도 부족했네요 🙀
YAML 파일에 지정한 port 변수는 보안 프로토콜의 포트 설정을 위한 값으로 TLS의 포트 번호가 담겨있었는데, 이 부분 수정해둘게요 🤍
